packages/myst-to-react/src/block.tsx

```diff
@@ -0,0 +1,77 @@
+import { Details } from './dropdown.js';
+import { MyST } from './MyST.js';
+import { SourceFileKind } from 'myst-spec-ext';
+import type { GenericParent } from 'myst-common';
+import classNames from 'classnames';
+import {
+  NotebookClearCell,
+  NotebookRunCell,
+  NotebookRunCellSpinnerOnly,
+  executableNodesFromBlock,
+} from '@myst-theme/jupyter';
```

Comment:
This can't be a dep of myst-to-react! Can you pull these together at the theme level instead.

A lot of lighter-weight integrations don't need jupyter, and that comes with a lot of overhead and integration main in some deployments.

i.e. the myst-to-react has a basic block renderer. Jupyter, if included by a theme has a more specific block renderer which takes priority.

Reply:
wherever this comes in leads to a patch-package in order just to build too :( 
Composing note renderers at theme level makes sense to me and is more scalable for core-myst functionality

Reply:
Yeah, I noticed the recursion when updating the deps!

I think this makes sense, thanks both!

Reply:
I've implemented a trivial block renderer (removing the Jupyter logic) in `myst-to-react`, and then moved the existing block renderer to `@myst-theme/jupyter`. I opted not to export the Jupyter renderer as part of the default exports; the `@myst-theme/site` package pulls these in for all themes, and based on this discussion I'm thinking that we might want to avoid that requirement. Perhaps in future we'll pull more renderer choices down to the themes.